client/css/style.css

```diff
@@ -1731,6 +1740,11 @@ button {
 	#viewport.menu-open {
 		transition: none;
 	}
+
+	/* Nick cycler should only be displayed on mobile */
```

Comment:
Actually, I don't really see any harm into having it available on desktop as well. Makes it easy to discover for people that are not used to the tab mechanic.